test/WebJobs.Extensions.DurableTask.Analyzers.Test/ActivityFunction/ArgumentAnalyzerTests.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Analyzers.Test.ActivityFunction
 {
-    [TestClass]
+    //[TestClass]
```

Comment:
Did you mean to leave this commented out? If so, could you add a code comment explaining why it's disabled?

Reply:
I can add a comment to this.